pkg/capabilities/generate_schemas/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/triggers"
+)
+
+func main() {
```

Comment:
Since this is a `cmd` can we put it in `cmd/capabilities/generate_schemas/main.go` or similar?

Reply:
Going to remove this file entirely, or rework it so that it regenerates all the test fixtures